sqlglot/parser.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import annotations
 
+import copy
```

Comment:
what are you using copy for?

Reply:
that's a leftover from playing around, removed